plugin/storage/es/mappings/fixtures/jaeger-dependencies.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Can we suffix these files with `-6` for consistency?

Reply:
done

Reply:
and fixtures too?